sections/shared/SystemStatus/SystemStatus.tsx

```diff
@@ -36,7 +39,7 @@ const SystemStatus: FC<SystemStatusProps> = ({ children }) => {
 		? isSystemOnMaintenanceQuery.data
 		: false;
 
-	return appOnMaintenance ? (
+	return appOnMaintenance || isL2 ? (
 		<>
 			<Head>
 				<title>{t('system-status.page-title')}</title>
```

Comment:
Probably the main content message should be updated to be more specific?

Reply:
Good idea. Thanks fixed